src/Mvc/src/Microsoft.AspNetCore.Mvc.Razor/Compilation/CompiledViewDescriptor.cs

```diff
@@ -37,59 +37,11 @@ public CompiledViewDescriptor(RazorCompiledItem item)
             RelativePath = ViewPath.NormalizePath(item.Identifier);
         }
 
-#pragma warning disable CS0618// Type or member is obsolete
-        /// <summary>
-        /// Creates a new <see cref="CompiledViewDescriptor"/>. At least one of <paramref name="attribute"/> or
-        /// <paramref name="item"/> must be non-<c>null</c>.
-        /// </summary>
-        /// <param name="item">The <see cref="RazorCompiledItem"/>.</param>
-        /// <param name="attribute">The <see cref="RazorViewAttribute"/>.</param>
-        public CompiledViewDescriptor(RazorCompiledItem item, RazorViewAttribute attribute)
```

Comment:
This was **just** marked as obsolete. I'm not sure why we'd remove it in 3.0 since it's not costing us anything to keep around.